### PR TITLE
Added subkey symbol-address to expose native pointers and typedefs.

### LINKED ISSUE #24: Add option to expose function pointers & typedefs
For some users it might be useful to expose the function pointers instead of Dart wrapper functions. In the use case where they want to pass these function pointers to C.

We should probably have a similar exclude/include mechanism specifying which functions should be exposed as function pointers.

### PR DESCRIPTION
**Pull request by mannprerak2**
Closes #24 
- Added new subkey `symbol-address` to expose native pointers and typedefs.
- Updated tests, changelog, version(`2.1.0 -> 2.2.0`).
- Added FAQ to readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -162,6 +162,10 @@ functions:
     'clang_dispose': 'dispose'
     # Removes '_' from beginning of a name.
     '_(.*)': '$1'
+  symbol-address:
+    # Used to expose native pointers and typedef.
```
Comment (dcharkes) [inline README.md:166]: Used to expose pointers to symbols and typedefs.

File: example/libclang-example/generated_bindings.dart [hunk 2]
```diff
@@ -6428,17 +6994,849 @@ class LibClang {
...
+  late final _clang_Type_visitFields_ptr =
+      _lookup<ffi.NativeFunction<Native_clang_Type_visitFields>>(
+          'clang_Type_visitFields');
+  late final _dart_clang_Type_visitFields _clang_Type_visitFields =
+      _clang_Type_visitFields_ptr.asFunction<_dart_clang_Type_visitFields>();
+
+  late final addresses = _SymbolAddress(this);
+}
+
+class _SymbolAddress {
```
Comment (dcharkes) [inline example/libclang-example/generated_bindings.dart:7013]: Maybe `_SymbolAddresses` because it holds all of them?

**Review by dcharkes — APPROVED** [review]

File: example/libclang-example/generated_bindings.dart (hunk 2 above)
Comment (mannprerak2) [reply example/libclang-example/generated_bindings.dart]: Done.

**Comment by dcharkes** [comment]
Sweet! 🚀

**Pull request merged**